File: src/main/java/it/gabrieletondi/telldontaskkata/useCase/SellItemRequest.java
package it.gabrieletondi.telldontaskkata.useCase;

public class SellItemRequest {
    private int quantity;
    private String productName;

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    public void setProductName(String productName) {
        this.productName = productName;
    }

    public int getQuantity() {
        return quantity;
    }

    public String getProductName() {
        return productName;
    }
}

File: typescript/src/useCase/SellItemRequest.ts
class SellItemRequest {
  private quantity: number;
  private productName: string;

  public setQuantity(quantity: number): void{
      this.quantity = quantity;
  }

  public setProductName(productName: string): void {
      this.productName = productName;
  }

  public getQuantity(): number {
      return this.quantity;
  }

  public getProductName(): string {
      return this.productName;
  }
}

export default SellItemRequest;
